Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/EconModulationMultiZone.mo

```diff
@@ -0,0 +1,194 @@
+  connect(retDamMinLimSig.y,retDamPos. x1) annotation (Line(points={{1,80},{2,80},
+          {40,80},{40,78},{58,78}}, color={0,0,127}));
+  connect(outDamMaxLimSig.y, outDamPos.x2) annotation (Line(points={{1,-50},{32,
+          -50},{32,-34},{58,-34}}, color={0,0,127}));
+  connect(TCooSet, damPosController.u_s) annotation (Line(points={{-140,10},{-140,
+          10},{-82,10}}, color={0,0,127}));
+  connect(uRetDamPosMin,retDamPos. f2)
+    annotation (Line(points={{-140,60},{-40,60},{-40,62},{58,62}}, color={0,0,127}));
+  connect(uOutDamPosMax, outDamPos.f2) annotation (Line(points={{-140,-70},{40,-70},
+          {40,-38},{50,-38},{58,-38}}, color={0,0,127}));
+  annotation (
+    defaultComponentName = "ecoMod",
+    Icon(graphics={
+        Rectangle(
+          extent={{-100,-100},{100,100}},
+          lineColor={0,0,127},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Line(points={{20,58}}, color={28,108,200}),
+        Line(
+          points={{-92,-84},{-50,-84},{12,70},{82,70}},
+          color={0,0,127},
+          thickness=0.5),
+        Line(
+          points={{-66,58},{12,58},{50,-76},{100,-76}},
+          color={0,0,127},
+          pattern=LinePattern.Dash,
+          thickness=0.5),
+        Text(
+          extent={{-108,138},{102,110}},
+          lineColor={0,0,127},
+          textString="%name")}),
+    Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-120,-120},{120,
+            120}}), graphics={
+        Rectangle(
+          extent={{-118,118},{18,-118}},
+          lineColor={0,0,0},
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{22,118},{118,-118}},
+          lineColor={0,0,0},
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid),
+                                   Text(
+          extent={{-84,56},{-40,16}},
+          lineColor={0,0,0},
+          fontSize=12,
+          horizontalAlignment=TextAlignment.Left,
+          textString="Damper position 
+supply air temperature
+control loop"),                    Text(
+          extent={{82,128},{126,88}},
+          lineColor={0,0,0},
+          fontSize=12,
+          horizontalAlignment=TextAlignment.Left,
+          textString="Damper position 
+assignments")}),
+    Documentation(info="<html>      
+    <p>
+    This is a multiple zone VAV AHU economizer modulation block. It calculates 
+    the outdoor and return air damper positions based on the supply air temperature
+    control loop signal. The implementation is in line with ASHRAE 
+    Guidline 36 (G36), PART5.N.2.c. Damper positions are linearly mapped to
+    the supply air control loop signal. This is a final sequence in the 
+    composite multizone VAV AHU economizer control sequence. Damper position 
+    limits, which are the inputs to the sequence, are the outputs of 
+    <code>EconDamperPositionLimitsMultiZone</code> and 
```

Comment:
Please add a hyperlink, using the full class name, e.g., `<a href=\"modelica://Buildings.Experimental. ...`